spec/unit/metadata_spec.rb

```diff
@@ -80,7 +80,9 @@ module Omnibus
       it_behaves_like 'a version manipulator', 'fedora', '11.5', '11'
       it_behaves_like 'a version manipulator', 'freebsd', '10.0', '10'
       it_behaves_like 'a version manipulator', 'gentoo', '2004.3', '2004.3'
+      it_behaves_like 'a version manipulator', 'ios_xr', '6.0.0.14I', '6'
       it_behaves_like 'a version manipulator', 'mac_os_x', '10.9.1', '10.9'
+      it_behaves_like 'a version manipulator', 'nexus', '5.0', '5'
```

Comment:
This looks suspect to me. Line 83 is dealing with the IOS XR release version (6.0.0.14I) rather than the WRL 7 release version (7.0.0.2), but line 85 is dealing with the WRL 5 release version (5.0) rather than the NX-OS release version (7.0(3)I2(1)). Which way should it be?